space-cli/cmd/modules/addons/database.go

```diff
@@ -232,8 +235,12 @@ func removeDatabase(alias string) error {
 		return nil
 	}
 
+	if err := deleteDatabaseConfig(project, alias); err != nil {
+		return err
+	}
+
```

Comment:
so that I could delete the database config from space-cloud as well. As of yet if we remove the database we cannot add it back bcz there still config in sc